src/ports/postgres/modules/svm/svm.py_in

```diff
@@ -676,27 +985,32 @@ def svm_help(schema_madlib, message, is_svc, **kwargs):
 def svm(schema_madlib, source_table, model_table,
         dependent_varname, independent_varname, kernel_func,
         kernel_params, grouping_col, params, is_svc,
-        verbose, detect_novelty=False, **kwargs):
+        verbose, add_intercept=False, **kwargs):
```

Comment:
Agreed, the kernel won't add an intercept even if the independent variables has a 1, unless `fit_intercept=true`, which is the default. The difference is that `array[1, ...]` is an intercept in feature space and `fit_intercept` creates intercept in kernel space.